.github/workflows/cve-scanning-gradle.yml

```diff
@@ -0,0 +1,25 @@
+name: CVE Scanning for Gradle
+
+on:
+  push:
+    paths:
```

Comment:
I would prefer this action to run on every push/pull_request if it lasts a reasonable duration.

Reply:
Also, the documentation mentions "dependencyCheckUpdate" task to update NVD from NIST. Do we need to add it to be sure the NVD is uptodate?

Reply:
Hi @symphony-soufiane fair question. I think for a GitHub action it is not needed as the action does not cache any build artifacts. 

As for the action to run on every push/pull/request. It seems it takes around ~6 min to complete. I have added the change to the action if you want to go ahead with it.